config/config.dist.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+/**
+ * Copy over the following configs to your config
+ */
+
+use SprykerEco\Shared\Econda\EcondaConstants;
+
+$config[EcondaConstants::ECONDA_CSV_FOLDER_PATH] = '';
+$config[EcondaConstants::ECONDA_CSV_DELIMITER] = '|';
```

Comment:
why is this an environment variable?

Reply:
it was an acceptance criteria

Reply:
You mean like that in DE environment it's a "|" but in AT it's ";" . It feels very strange. Could you please point me out to the acceptance criteria or PM ?

Reply:
Talked to the Alex, and we decide to move this configuration to the constant